crates/astria-conductor/src/executor/mod.rs

```diff
@@ -59,6 +58,8 @@ use state::StateReceiver;
 
 use self::state::StateSender;
 
+type CelestiaHeight = u64;
```

Comment:
note: doing this because it makes the update enums much more clear or what is being passed in.